main.go

```diff
@@ -60,6 +60,10 @@ func (l *Linter) String() string {
 func LinterFromName(name string) *Linter {
 	s := linterDefinitions[name]
 	parts := strings.SplitN(s, ":", 2)
+	if len(parts) < 2 {
+		kingpin.FatalUsage("invalid linter: %q", name)
```

Comment:
In general I prefer not to do that as it makes the "fast path" of dealing with errors slower, because the usage output is so long that the actual error scrolls off the top of the terminal. Additionally, `Fatalf()` appends `try --help`, so the user can find more information if they desire.